addons/web/static/src/views/list/list_renderer.xml

```diff
@@ -221,7 +221,10 @@
                         <t t-foreach="groupByButtons[group.groupByField.name]" t-as="button" t-key="button.id">
                             <t t-if="!evalInvisible(button.invisible, group.record)">
                                 <t t-if="button.clickParams.type === 'edit'">
-                                    <button t-att-title="button.title" class="btn" t-on-click.stop="() => this.editGroupRecord(group)" tabindex="-1">
+                                    <button t-if="button.string" t-att-title="button.title" class="btn btn-secondary" t-on-click.stop="() => this.editGroupRecord(group)" tabindex="-1">
+                                        <t t-if="button.string" t-esc="button.string"/>
+                                    </button>
+                                    <button t-else="" t-att-title="button.title" class="btn" t-on-click.stop="() => this.editGroupRecord(group)" tabindex="-1">
```

Comment:
I think that we shouldn't even have allowed to tweak the icon in the first place. The design of this "special" button should be enforced by the framework for the sake of consistency.  But anyway, if you want this change, could you add a tiny unit test?

Reply:
Then it should probably be a "View" button everywhere, at least according to FP

Reply:
> Then it should probably be a "View" button everywhere, at least according to FP

I agree.

Reply:
I added a test for both cases